src/components/App/index.tsx

```diff
@@ -75,7 +75,7 @@ export class AppBase extends React.Component<Props> {
         <Route
           component={Browse}
           exact
-          path="/:lang/:application/files/browse/:versionId/"
+          path="/:lang/browse/:addonId/versions/:versionId/"
```

Comment:
Is the existing DevHub going to switch to this URL? It still looks like the old URL on dev. I believe we need to keep the URLs in sync.

Reply:
We discussed this during a call. We can use any URL because it is internal only, and we will never share this URL publicly. We agreed on changing the URL, removing the `application` parameter because it was not needed.

That being said, we did not change the URL in DevHub because we did not deploy this project on an hosted env yet. The old/current code review tool is served and linked in the -dev DevHub.

Last but not least, the API expects an `addonId` now so we have to have one in the URL, but this is a new behavior, different than the DevHub (which does not rely on the API). That's why I went ahead and updated this URL. We'll have to use this URL in DevHub when this project will be deployed on an hosted env and we won't be shy to actually link to it in DevHub.